### DISCUSSION TITLE
Set window title based on active tab

### DISCUSSION THREAD

**Discussion by mlncn** [linuxdeepin/developer-center#4811] [Development Questions | 开发答疑]
Disclosure: Using Deepin Terminal in Gnome, so maybe all this Just Works in DDE.

My window switcher lets me type a few characters to narrow the results based on the window title.  ALT+TAB+dee filters my dozen browser windows down to this one, for example.

If Deepin Terminal windows inherited their title from the active tab, this (and probably many other uses, like showing all windows at once) would work delightfully.

**Comment by BLumia** [comment]
Seems reasonable :)

cc @ArchieMeng
